coral/src/app/main.tsx

```diff
@@ -46,7 +46,7 @@ const queryClient = new QueryClient({
         }
         return failureCount < 2;
       },
-      refetchOnWindowFocus: false,
+      refetchOnWindowFocus: true,
```

Comment:
I think we should do `refetchOnWindowFocus` only for the query for request 🤔  Otherwise it's a bit much maybe?

Reply:
Yeah, I don't know :o I feel like a tool like Klaw stays open in the background a lot, and due to its nature, there may have been many changes since you last focused the window... 

But it *will* flash loading states for everything that does not have `keepPreviousData`. 

I think we can add it to individual queries in these circumstances, as you said :o

Reply:
>I feel like a tool like Klaw stays open in the background a lot, and due to its nature, there may have been many changes since you last focused the window... 

Yeah, that's true. I think we add it for now to individual queries and then it would be really nice to get some insights how users use it :D (ok that's really hard) to see if actively reloading a lot is beneficial 🤔   But for now I would not refetch eagerly so much, because we've some more heavy endpoints and a lot of them. So having more control on them feels a bit better to me.

But with notifications refetching often we get the most important information to users I feel! We could even think about refreshing that on an interval in the background (but not for this PR).